README.md

```diff
@@ -1,2 +1,282 @@
-# portfolio
-microverse mobile first portfolio
+<a name="readme-top"></a>
+
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
+
+Modify this file to match your project and remove sections that don't apply.
+
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
+
+OPTIONAL SECTIONS:
+- FAQ
+
+After you're finished please remove all the comments and instructions!
```

Comment:
- [ Optional ] My suggestion is to take this section of the comment out, as its a guide for you, but should not be committed

Reply:
Fixed.